sechub-shared-kernel/src/main/java/com/daimler/sechub/sharedkernel/usecases/admin/user/UseCaseAdministratorAssignsOwnerToProject.java

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+package com.daimler.sechub.sharedkernel.usecases.admin.user;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.daimler.sechub.sharedkernel.Step;
+import com.daimler.sechub.sharedkernel.usecases.UseCaseDefinition;
+import com.daimler.sechub.sharedkernel.usecases.UseCaseGroup;
+import com.daimler.sechub.sharedkernel.usecases.UseCaseIdentifier;
+
+/* @formatter:off */
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+@UseCaseDefinition(
```

Comment:
As mentioned before: The usecase and text parts should be more like "UseCaseAdministratorChangesProjectOwner" or something like that.

Reply:
done